server/store/sqlstore.go

```diff
@@ -0,0 +1,75 @@
+package store
+
+import (
+	"database/sql"
+	"net/url"
+
+	"github.com/mattermost/mattermost/server/public/plugin"
+	"github.com/mattermost/mattermost/server/public/pluginapi/cluster"
+
+	"github.com/mattermost/mattermost-plugin-user-survey/server/model"
+)
+
+const (
+	TablePrefix = "user_survey_"
```

Comment:
Just curious - why is this configurable? Will we find ourselves needing to change this at some point? Or is it so that people can whitelabel the plugin?

Reply:
We'll use a different prefix in tests, for example. But other than that, its just here instead of being hardcoded, kinda like a central place.